v2/sunshine/user_profiles_app/assets/index.html

```diff
@@ -93,6 +93,7 @@
         })
       }
       keys.forEach(key => {
+        if (key === 'current_location' || key === 'role_id') return;
```

Comment:
so i'm thinking the best way to implement this would be to keep a running list of the keys we want to filter out, then filter our list against those set of keys. this way we can add new keys the backend may add to this running list
```
// global scope
const FILTER_OUT_ATTRIBUTES = new Set([
  'current_location',
  'role_id',
])

// down here
keys.filter(key => !FILTER_OUT_ATTRIBUTES.has(key)).forEach(key => {
// ..etc
```

Reply:
actually better yet, let's do this filter on the line where `keys` is defined
```
const keys = Object.keys(object).filter(key => !FILTER_OUT_ATTRIBUTES.has(key))
```